### PR TITLE
Updates to rdma-code

### PR DESCRIPTION
**Pull request by amadio**
@Zlogene Please review. Thanks!

### CODE REVIEW

File: sys-cluster/rdma-core/rdma-core-9999.ebuild [hunk 1]
```diff
@@ -33,7 +33,7 @@ COMMON_DEPEND="
 DEPEND="${COMMON_DEPEND}
 	python? (
 		$(python_gen_cond_dep '
-			dev-python/cython[${PYTHON_USEDEP}]
+			dev-python/cython[${PYTHON_MULTI_USEDEP}]
```
Comment (thesamesam) [inline sys-cluster/rdma-core/rdma-core-9999.ebuild:36]: That’s the obsolete one :)

File: sys-cluster/rdma-core/rdma-core-9999.ebuild [hunk 2]
```diff
@@ -56,42 +56,34 @@ RDEPEND="${COMMON_DEPEND}
...
-		-DCMAKE_INSTALL_UDEV_RULESDIR="$(get_udevdir)"/rules.d
+		-DCMAKE_INSTALL_UDEV_RULESDIR="${EPREFIX}""$(get_udevdir)"/rules.d
 		-DCMAKE_INSTALL_SYSTEMD_SERVICEDIR="$(systemd_get_systemunitdir)"
-		-DCMAKE_DISABLE_FIND_PACKAGE_pandoc=yes
-		$(ver_test -ge 25 && echo -DCMAKE_DISABLE_FIND_PACKAGE_rst2man=yes)
 		-DCMAKE_DISABLE_FIND_PACKAGE_Systemd="$(usex systemd no yes)"
 		-DENABLE_VALGRIND="$(usex valgrind)"
 		-DENABLE_RESOLVE_NEIGH="$(usex neigh)"
 		-DENABLE_STATIC="$(usex static-libs)"
+		-DNO_PYVERBS="$(usex python OFF ON)"
```
Comment (thesamesam) [inline sys-cluster/rdma-core/rdma-core-9999.ebuild:75]: I’d consider doing usex !python but it’s up to you.

File: sys-cluster/rdma-core/rdma-core-9999.ebuild [hunk 3]
```diff
@@ -3,7 +3,7 @@
 
 EAPI=7
 
-PYTHON_COMPAT=( python3_8 )
+PYTHON_COMPAT=( python3_{6..10} )
```
Comment (thesamesam) [inline sys-cluster/rdma-core/rdma-core-9999.ebuild:6]: 3.6 and 3.7 are ignored by the eclass

File: sys-cluster/rdma-core/rdma-core-9999.ebuild (hunk 1 above)
Comment (amadio) [reply sys-cluster/rdma-core/rdma-core-9999.ebuild:36]: Oops, merged after an ack from zlogene on IRC... Will fix after the fact.

File: sys-cluster/rdma-core/rdma-core-9999.ebuild (hunk 3 above)
Comment (amadio) [reply sys-cluster/rdma-core/rdma-core-9999.ebuild:6]: Same as above.

File: sys-cluster/rdma-core/rdma-core-9999.ebuild (hunk 2 above)
Comment (amadio) [reply sys-cluster/rdma-core/rdma-core-9999.ebuild:75]: Good idea, will change that.

**Comment by amadio** [comment]
Merged manually, closing. Thanks for the comments, @thesamesam, I will apply them after the fact.

**Pull request closed without merging**